test/tests/unit/probe/router.test.ts

```diff
@@ -819,5 +819,26 @@ describe('probe router', () => {
 
 			expect(probes.length).to.equal(0);
 		});
+
+		it('should return match for user tag', async () => {
```

Comment:
Let's add a test for the part that user tags are not used in magic mode.

Reply:
Since router.test.ts works with already build sockets I've added a new test to the create-measurement.test.ts + building is covered in a new fetch-sockets.test.ts